### DISCUSSION TITLE
Version 1.0 Functional Equivalence

### DISCUSSION THREAD

**Discussion by ballsteve** [ballsteve/xrust#90] [Announcements]
This release of χrust has achieved functional equivalence with v1.0 of XPath and XSLT. That is, all of the elements and functions in v1.0 have been implemented. Please note that χrust is not compliant with v1.0 of XPath/XSLT, since it implements the v3.0 data model.

<hr /><em>This discussion was created from the release <a href='https://github.com/ballsteve/xrust/releases/tag/release_1_0_0'>Version 1.0 Functional Equivalence</a>.</em>

**Comment by Devasta** [comment]
Fantastic work, delighted to see this day come. 😄

**Reply by ballsteve** [reply]
Yes, it's been a long time coming ;-) I'm pretty pleased with how the parser has evolved, you've done great work there!

**Comment by Devasta** [comment]
And plenty more to do! 😄

Need anything done to make the release available on Cargo?

**Comment by ballsteve** [comment]
v1.0.0 is now published to crates.io
